frameworks/hdfs/src/main/dist/core-site.xml

```diff
@@ -32,7 +32,12 @@
         <name>ha.zookeeper.parent-znode</name>
         <value>/{{SERVICE_ZK_ROOT}}/hadoop-ha</value>
     </property>
-
+    {{#TLS_ENABLED}}
+    <property>
+        <name>hadoop.ssl.hostname.verifier</name>
+        <value>ALLOW_ALL</value>
```

Comment:
It can be templated. Other values are:
- DEFAULT
- STRICT
- STRICT_I6
- DEFAULT_AND_LOCALHOST 
- ALLOW_ALL